airbyte-workers/src/main/java/io/airbyte/workers/process/KubePodProcess.java

```diff
@@ -653,7 +653,8 @@ private int getReturnCode(final Pod pod) {
       }
       // If the pod cannot be found and was not killed, it either means 1) the pod was not created
       // properly 2) this method is incorrectly called.
-      throw new RuntimeException("Cannot find pod while trying to retrieve exit code. This probably means the Pod was not correctly created.");
+      throw new RuntimeException(
+          String.format("Cannot find pod (%s) while trying to retrieve exit code. This probably means the pod was not correctly created.", name));
```

Comment:
It may also be helpful to include the namespace in this message, to make it crystal clear what this was looking for